src/com/elikill58/negativity/spigot/listeners/NegativityPacketEvent.java

```diff
@@ -5,14 +5,15 @@
 import org.bukkit.event.HandlerList;
 
 import com.elikill58.negativity.spigot.SpigotNegativityPlayer;
-import com.elikill58.negativity.spigot.packets.ReceivedPacket;
+import com.elikill58.negativity.spigot.packets.AbstractPacket;
 
+@Deprecated
```

Comment:
Il faudrait indiquer dans le javadoc de la classe ce qui la remplace avec un tag `@deprecated`
https://docs.oracle.com/javase/7/docs/technotes/guides/javadoc/deprecation/deprecation.html#javadoc_tag

Reply:
That's a good idea, i did it https://github.com/Elikill58/Negativity/pull/121/commits/885a730873b714a24cacf1d299454883e557344f